styleguide/components/31-molecules/checkbox-with-filter/README.md

```diff
@@ -9,11 +40,6 @@
   a filter field with counter and a collection of
   checkbox components (.checkbox-filter__checkboxes).
 
-## Functionality
-
-* Checked items appear in an overview outside of the modal.
-* Closing the modal without confirmation restores the previous state.
-
 ## Accessibility
 
 * The modal uses a tabtrap.
```

Comment:
I should really expand this some more.
Could you hold the PR open? I'll look into it tomorrow.

Reply:
@delrueba The functionality is already explained as it should. If you want to add extra information about implementation or accessibility, please do, but maybe don't do it on this PR. Documentation is something that is constantly improving and will "never be finished" so you dan do this in a new PR. Otherwise changes will never go through :)